### PR TITLE
Synthflesh cloning port

### PR DESCRIPTION
**Pull request by TheNeoGamer42**
## About The Pull Request
Ports https://github.com/WaspStation/WaspStation-1.0/pull/55

ROUNDSTART CLONERS START WITH (some) SYNTHFLESH, SO DON'T WORRY TOO MUCH!
You now need to add synthflesh to the cloning pod to clone people.

t1 parts -> 90u/clone
t4 parts -> ~13u/clone

image
image

## Why It's Good For The Game
Balances cloning without removing it, unlike /tg/.

## Changelog
:cl:
tweak: Cloners now need some synthflesh to operate.
add: Clonepods now have 100u of synthflesh in them at roundstart
tweak: Roundstart cloning is now 100% speed instead of 200% (slower cloning roundstart)
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Looks like some extra map changes snuck in

**Comment by TheNeoGamer42** [comment]
That honestly confuses me, all I did was open strongDMM and place 2 beakers of synthflesh in cloning rooms, not sure how anything else would change.

**Comment by TheNeoGamer42** [comment]
okay it still says theres alot of map differences, so apparently my base version of beecode has the differences too, soooooooo.....

**Comment by TheNeoGamer42** [comment]
ok strongDMM fucked up the maps somehow I guess

**Comment by TheNeoGamer42** [comment]
Tested everything on my end, seems to work perfectly fine. No strange map changes from the void and cloners have 100u roundstart thats not in beakers, atleast on my local test

**Comment by qwertyquerty** [comment]
Not a fan of the path /mapped do you think you could find a better name for it?

Like /prefilled or anything better you can think of

**Comment by TheNeoGamer42** [comment]
honestly I have no clue how to change the path of the filled one and mark yelled at me to use the /mapped instead of throwing 2 beakers in there

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -524,6 +568,12 @@
 
 	flesh_number = unattached_flesh.len
 
+/obj/machinery/clonepod/mapped
+
+/obj/machinery/clonepod/mapped/Initialize()
+	. = ..()
```
Comment (Crossedfall) [inline code/game/machinery/cloning.dm]:
```suggestion
/obj/machinery/clonepod/prefilled

/obj/machinery/clonepod/prefilled/Initialize()
	. = ..()
```

**Review by Crossedfall — CHANGES_REQUESTED** [review]
You need to update the code where it's initialized and then ensure that the maps are updated as well. You could do a quick find and replace on the maps in this instance since it's a really basic line change.

I'll provide a suggestion on the initialization to help you get started.

File: code/game/machinery/cloning.dm (hunk 1 above)
Comment (TheNeoGamer42) [reply code/game/machinery/cloning.dm]: did that and it seems to work, even checked varedit in a test so its now that instead of /mapped

**Review by Crossedfall — APPROVED** [review]
seems good to me

**Pull request merged**